capstone/src/main/java/com/step902020/capstone/EventController.java

```diff
@@ -74,12 +73,16 @@
         noPastEvents.add(e);
       }
     }
+
+    // order by rank
+    Collections.sort(noPastEvents, (a, b) -> Integer.compare(b.getRank(), a.getRank()));
```

Comment:
Is it possible to rely on Datastore to do the sort when you query for events above?

Reply:
No its not. find by example does not support any other form of concatenated fields. I was't able to add "OrderByRankDesc"

Reply:
Have you tried adding a `Sort` argument instead?
Something like:
```
Iterable<Event> findAll(Example<Event> example, Sort sort);
```

Reply:
No I have not I can definitely try it.